### PR TITLE
feat:slider

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**

### CODE REVIEW

File: Components/Rating/rating.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+}
+
+.rating-box input {
+    display: none;
+}
+
+
+.rating-box label{
+    color: #ccc;
```
Comment (som749898) [inline Components/Rating/rating.css]: use css variable for color

File: Components/Slider/Slider.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+.slidecontainer{
+    width: 100%;
+    
+}
+
+
+
+
```
Comment (som749898) [inline Components/Slider/Slider.css:11]: you can remove extra spaces here

**Review by som749898 — COMMENTED** [review]
Have left few suggestion
Rest looks good to merge

File: Components/Rating/rating.css (hunk 1 above)
Comment (HarshithaKonduru214) [inline Components/Rating/rating.css]: Use variables for colors.

**Review by HarshithaKonduru214 — APPROVED** [review]

**Pull request merged**